code/game/machinery/Sleeper.dm

```diff
@@ -158,7 +158,8 @@
 	return attack_hand(user)
 
 /obj/machinery/sleeper/attackby(var/obj/item/I, var/mob/user)
-	add_fingerprint(user)
+	if(!istype(I, /obj/item/forensics))
```

Comment:
Considering how much this is used, this should probably be an atom flag. Or just put the check in add_fingerprint tbh.